### ISSUE TITLE
Threshold/dead zone on touch controls

### ISSUE THREAD

**Issue by ThatIsAPseudo** [TeamNewPipe/NewPipe#5606]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
There could be a threshold or some sort of dead zone when you touch the screen, so that it does not immediatly change the brightness/volume. Moreover the current volume/brightness should be displayed while in the dead zone so you know that if you move an inch more you'll trigger slider.

Or, the slider speed could be slower so that only a small volume/brightness change would be made.

#### Is your feature request related to a problem? Please describe it

I often find myself wrongly changing the brightness or volume of a video juste because I wanted to wipe some dust from my screen. Generally, I have to keep in mind that I *must not* touch my screen while I'm watching a video since I can mess something (especially when it's dark or when I need to be quiet).

#### How will you/everyone benefit from this feature?

Unintended sudden volume or brightness changes can screw up the user experience, whereas adding a threshold zone or slowing down a bit the slider won't harm the flow.

**Comment by SameenAhnaf** [comment]
>I often find myself wrongly changing the brightness or volume of a video juste because I wanted to wipe some dust from my screen.

@mhmdanas If I am not wrong, locking touch controls of screen is a lot more logical than manually choosing a dead zone. So, this issue could be closed in favor of #5740.

@ThatIsAPseudo You may check the APK in #5799.

**Comment by ThatIsAPseudo** [comment]
@SameenAhnaf Indeed, locking touch controls could do the job, thanks for pointing this out
By reading myself again I realize I wasn't very clear though ; what I'd like is less sensitive touch controls, without losing the ability to use it.
